src/Tizen.NUI/src/internal/Application.cs

```diff
@@ -453,74 +453,77 @@ protected override void Dispose(DisposeTypes type)
 
             if (_applicationInitEventCallbackDelegate != null)
             {
-                ApplicationSignal initSignal = this.InitSignal();
-                initSignal?.Disconnect(_applicationInitEventCallbackDelegate);
-                initSignal?.Dispose();
+                _initSignal?.Disconnect(_applicationInitEventCallbackDelegate);
```

Comment:
in c#, pascal case is supposed to be used and for private member, it would be OK to use `_` prefix. However in NUI, we have decided not to use prefix such as `_`, `m`, `p`, and etc.

- NUI ground rule : https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule
- NUI coding rule : https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=191373542

could you use PascalCase even for private fields?

Reply:
Done. Please check again.
Thanks